styletesla.css

```diff
@@ -1,79 +1,286 @@
 .body {
-    max-width: 1440px;
+	max-width: 1440px;
     margin: 0 auto;
     }
 .mainpage {
 	height: 100vh;
-        }
+    }
 .header {
 	height: 100px;
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-}
+	margin-bottom: 40px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5505628)